middle_end/flambda2/types/flambda_type.mli

```diff
@@ -227,27 +227,15 @@ val join :
 type 'a type_accessor = Typing_env.t -> 'a
 
 module Function_declaration_type : sig
-  module Inlinable : sig
+  module T0 : sig
```

Comment:
I'd really like it if we could find a better name for this module: as it stands, the name ` T0` is very uninformative about what it is.

Reply:
See above